ca-cwds/CALS
# CALS 5575- Facility profile header

#### Which User story does this relate to?
- https://osi-cwds.atlassian.net/browse/CALS-5575

#### Brief feature description
-  Add a back button on facility profile page header. The back button should retain everything on the search results page.

How to test?
Go to Search -> click search -> click on any facility that are present in the results -> go to the clicked facility page -> click on the back button -> everything should be retained as it is

#### Does this PR block/effect other stories?
- no

#### Do any commands needs to be run in order to run the project with this PR?
-  yarn install

#### Code coverage percent diff as a result of this PR?
-
